03-task-manager/starter/quizAnswers.txt

```diff
@@ -0,0 +1,13 @@
+Quickly explain how middleware is used in an Express application.
+What does CRUD stand for?
+    Middleware is used for request, response and next methods that help with data being 
+    moved back and forth. CRUD is create, read, update, and delete
```

Comment:
Nitpick:
For middleware, you can think of it as functions that can be used to perform tasks like parsing incoming requests, adding headers, handling authentication, and handling errors. In other words, it's pre-processing a request with some logic before it reaches the intended route.